hooks/common/post-code-update/post-code-update.sh

```diff
@@ -40,51 +40,64 @@ if [[ $target_env =~ ^ode\d* ]]; then
   target_env="ode";
 fi
 
-## Clear last install out of Memcache.
-## Drush CR will not work on a non-existant site. Drush CC will not find the memcached
-## backed bins if the site is not installed. So we made our own to work with empty sites
-## to clean out memcache. (DB backing stores disappear as soon as the DB is dropped.)
-drush cgov:destroy-cache
+# This variable must be the word true for it to update instead of reinstall
+if [[ $PRESERVE_ON_REDEPLOY == "true" ]]; then
+  ## Just do a drupal update and update translations
+  blt deploy:update --environment=$target_env -v --no-interaction -D drush.ansi=false
 
-## Perform a fresh install.
-blt artifact:install:drupal --environment=$target_env -v --no-interaction -D drush.ansi=false
+  ## Reload translation pack.
+  blt cgov:locales:translate -D drush.ansi=false
 
-## Load our test users.
-blt cgov:user:load-all -D cgov.drupal_users_file=$users_file -D drush.ansi=false
+  ## Setup some default JS globals.
+  cat FrontendGlobals.json | drush config:set cgov_core.frontend_globals config_object -
+else
+  ## Clear last install out of Memcache.
+  ## Drush CR will not work on a non-existant site. Drush CC will not find the memcached
+  ## backed bins if the site is not installed. So we made our own to work with empty sites
+  ## to clean out memcache. (DB backing stores disappear as soon as the DB is dropped.)
+  drush cgov:destroy-cache
 
-## Reload translation pack.
-blt cgov:locales:translate -D drush.ansi=false
+  ## Perform a fresh install.
+  blt artifact:install:drupal --environment=$target_env -v --no-interaction -D drush.ansi=false
 
-## Setup some default JS globals.
-cat FrontendGlobals.json | drush config:set cgov_core.frontend_globals config_object -
```

Comment:
The old code reloaded translations after loading the users.  The equivalent code in the `else` block has reversed those  two steps.

I do not know if that is a significant difference.